index.html

```diff
@@ -2,8 +2,7 @@
 layout: default
 class: home
 hide_breadcrumbs: true
-title: Choosing an OSS license doesn’t need to be scary
-description: A site to provide non-judgmental guidance on choosing a license for your open source project
+title: Choosing an open source license doesn’t need to be scary
```

Comment:
By the same token, you might consider changing the `<h1>` on the index page similarly from:

![image](https://cloud.githubusercontent.com/assets/121322/10706580/d2040194-799c-11e5-8403-9a8ad2939a1d.png)

to

![image](https://cloud.githubusercontent.com/assets/121322/10706609/0514d96e-799d-11e5-9834-e6207ed7896e.png)

The former definitely *looks* nicer.